### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by mxmilkiib** [sjaehn/BJumblr#8]
Getting this atm;

```
Build BJumblr.lv2 DSP...src/BJumblr.cpp: In constructor ‘BJumblr::BJumblr(double, const LV2_Feature* const*)’:
src/BJumblr.cpp:70:21: error: ‘invalid_argument’ is not a member of ‘std’
   70 |  if (!m) throw std::invalid_argument ("BJumblr.lv2: Host does not support urid:map.");
      |                     ^~~~~~~~~~~~~~~~
src/BJumblr.cpp:71:34: error: ‘invalid_argument’ is not a member of ‘std’
   71 |  if (!workerSchedule) throw std::invalid_argument ("BJumblr.lv2: Host does not support work:schedule.");
      |                                  ^~~~~~~~~~~~~~~~
src/BJumblr.cpp: In constructor ‘BJumblr::Sample::Sample(const char*)’:
src/BJumblr.cpp:104:50: error: ‘invalid_argument’ is not a member of ‘std’
  104 |         if (!sndfile || !info.frames) throw std::invalid_argument("BJumblr.lv2: Can't open " + std::string(samplepath) + ".");
      |                                                  ^~~~~~~~~~~~~~~~
src/BJumblr.cpp: In member function ‘LV2_State_Status BJumblr::state_restore(LV2_State_Retrieve_Function, LV2_State_Handle, uint32_t, const LV2_Feature* const*)’:
src/BJumblr.cpp:701:15: error: ‘invalid_argument’ in namespace ‘std’ does not name a type
  701 |   catch (std::invalid_argument &ia)
      |               ^~~~~~~~~~~~~~~~
src/BJumblr.cpp:703:29: error: ‘ia’ was not declared in this scope
  703 |    fprintf (stderr, "%s\n", ia.what());
      |                             ^~
src/BJumblr.cpp: In member function ‘LV2_Worker_Status BJumblr::work(LV2_Worker_Respond_Function, LV2_Worker_Respond_Handle, uint32_t, const void*)’:
src/BJumblr.cpp:842:17: error: ‘invalid_argument’ in namespace ‘std’ does not name a type
  842 |     catch (std::invalid_argument &ia)
      |                 ^~~~~~~~~~~~~~~~
src/BJumblr.cpp:844:31: error: ‘ia’ was not declared in this scope
  844 |      fprintf (stderr, "%s\n", ia.what());
      |                               ^~
make: *** [makefile:109: BJumblr.so] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
Error making: bjumblr.lv2-git
```

**Comment by sjaehn** [comment]
Thank you for information. Should be fixed in f73e7e812b3880713d2efcbf823ae45a1776b9c0 . Please tell me if the problem persists.

**Comment by mxmilkiib** [comment]
That has resolved it, thanks :)
